deepinv/datasets/div2k.py

```diff
@@ -30,81 +50,70 @@ class DIV2K(torch.utils.data.Dataset):
-        if self._check_dataset_exists():
-            print(
-                f"""
-            Dataset is available at `{self.root}`.
-            `download` flag is not taken into account.
-            """
-            )
-        # otherwise we try to download the whole dataset
-        elif download:
-            if not os.path.isdir(self.root):
-                os.makedirs(self.root)
-            if os.path.exists(train_dir_path):
-                raise ValueError(
-                    f"""
-                The train folder already exists,
-                thus the download is aborted.
-                Please set `download=False`
-                OR remove `{train_dir_path}`."""
-                )
-
-            if os.path.exists(valid_dir_path):
-                raise ValueError(
-                    f"""
-                The val folder already exists,
-                thus the download is aborted.
-                Please set `download=False`
-                OR remove `{valid_dir_path}`."""
-                )
-
-            for filename, url in self.zipfile_urls.items():
-                # download zipfile from the Internet and save it locally
-                download_zipfile(
-                    url=self.zipfile_urls[filename],
-                    save_path=os.path.join(self.root, filename),
-                )
-                # extract local zipfile
-                extract_zipfile(os.path.join(self.root, filename), self.root)
-        # stop the execution since the dataset is not available and we didn't download it
-        else:
-            raise RuntimeError(
-                f"""
-            Dataset not found at `{self.root}`.
-            Please set `root` correctly (currently `root={self.root}`),
-            AND check that `{train_dir_path}` contains ONLY the following files 0001.png, ..., 0800.png
-                           `{valid_dir_path}` contains ONLY the followinf files 0801.png, ..., 0900.png
-            OR set `download=True` (currently `download={download}`).
-            """
-            )
 
         if self.mode == "train":
-            self.img_dir = train_dir_path
+            self.img_dir = os.path.join(self.root, "DIV2K_train_HR")
         elif self.mode == "val":
-            self.img_dir = valid_dir_path
+            self.img_dir = os.path.join(self.root, "DIV2K_valid_HR")
         else:
             raise ValueError(
                 f"Expected `train` or `val` values for `mode` argument, instead got `{self.mode}`"
             )
 
+        # download a split of the dataset, we check first that this split isn't already downloaded
+        if not self._verify_split_dataset_integrity():
+            if download:
+                if not os.path.isdir(self.root):
+                    os.makedirs(self.root)
+                if os.path.exists(self.img_dir):
+                    raise ValueError(
+                        f"""
+                    The {self.mode} folder already exists,
```

Comment:
can you print all the string in one line, without any space at the beginning?

Reply:
Done :
every multi line strings  ->  one-line string